hudi-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -316,43 +320,17 @@ public void testMultipleAppend(HoodieLogBlockType dataBlockType) throws IOExcept
     }, "getCurrentSize should fail after the logAppender is closed");
   }
 
-  /*
-   * This is actually a test on concurrent append and not recovery lease. Commenting this out.
-   * https://issues.apache.org/jira/browse/HUDI-117
-   */
-
-  /**
-   * @Test public void testLeaseRecovery() throws IOException, URISyntaxException, InterruptedException { Writer writer
```

Comment:
as the comment mentions, this testcase is not as the name suggests, also the logic is covered in other test case, hence removing it